MiniApp/Classes/Protocols/MiniAppMessageProtocol.swift

```diff
@@ -19,6 +19,18 @@ public protocol MiniAppMessageProtocol: class {
     func shareContent(info: MiniAppShareContent, completionHandler: @escaping (Result<MASDKProtocolResponse, Error>) -> Void)
 }
 
+/**
+ Public Protocol that will be used by the Mini App to communicate
+ with the Native implementation for User profile related retrieval
+*/
+public protocol MiniAppUserInfoProtocol: class {
```

Comment:
this should be renamed as MiniAppUserInfoDelegate to conform to the delegation pattern best practice used in Swift

Reply:
Done